### DISCUSSION TITLE
Multiple URLs

### DISCUSSION THREAD

**Discussion by predmijat** [go-shiori/shiori#583] [Q&A]
Hi,

Sometimes I'd like to save two URLs for a single bookmark (e.g. the "real" URL and URL for discussion on HN or similar). Am I missing the obvious way to that with shiori?

Thanks!

**Comment by predmijat** [comment]
Any suggestions about this? Maybe something like "notes" for each bookmark or similar...?
Edit: so one idea is to use excerpt, but it would be great if we can get https://github.com/go-shiori/shiori/discussions/535#discussion-4559303 sorted.

**Reply by fmartingr** [reply]
Hey @predmijat, I'd say that for your use case a new "notes" field would be the ideal approach, where you can put not only URLs but anything related to the bookmark, depending on the situation.
